### PR TITLE
Update Web API migration doc to RTM

### PR DESCRIPTION
**Pull request by scottaddie**
Upgrade the sample code for the "Migrating from ASP.NET Web API" doc from RC1 to RTM. Also, adjust a few minor details in the doc accordingly.

### CODE REVIEW

**Comment by scottaddie** [comment]
@spboyer Any idea what might be causing the build to fail? I see the following error, but I'm not able to make sense of what's wrong with my syntax on line 72:

"/home/travis/build/aspnet/Docs/aspnet/migration/webapi.rst:72: WARNING: Could not lex literal_block as "c#". Highlighting skipped."

**Comment by spboyer** [comment]
@scottaddie the syntax is 
```
:language: csharp
```

File: aspnet/migration/webapi.rst [hunk 1]
```diff
@@ -69,18 +67,18 @@ Delete the ``Project_Readme.html`` file from the new project. Your solution shou
...
 ---------------------
 
-ASP.NET Core no longer uses *Global.asax*, *web.config*, or *App_Start* folders. Instead, all startup tasks are done in *Startup.cs* in the root of the project (see :doc:`/fundamentals/startup`). In ASP.NET Core MVC attribute-based routing is now included by default when ``UseMvc()`` is called and this is the recommended approach for configuring Web API routes (and is how the Web API starter project handles routing).
+ASP.NET Core no longer uses *Global.asax*, *web.config*, or *App_Start* folders. Instead, all startup tasks are done in *Startup.cs* in the root of the project (see :doc:`/fundamentals/startup`). In ASP.NET Core MVC, attribute-based routing is now included by default when ``UseMvc()`` is called; and, this is the recommended approach for configuring Web API routes (and is how the Web API starter project handles routing).
 
 .. literalinclude:: webapi/sample/ProductsCore/Startup.cs
   :language: c#
-  :emphasize-lines: 43
+  :emphasize-lines: 40
   :linenos:
```
Comment (spboyer) [inline aspnet/migration/webapi.rst]: remove the `:linenos:`

**Comment by spboyer** [comment]
@scottaddie if you can make these few changes, should be good.

**Comment by scottaddie** [comment]
@spboyer Thanks for taking a look, but that didn't seem to fix the problem (at least not locally). Wouldn't the build have failed earlier in the file if that `:language: c#` syntax was incorrect?

**Comment by spboyer** [comment]
I'll pull the branch and see what the issue is.

**Comment by scottaddie** [comment]
@spboyer I really appreciate you taking the time to do this. Thank you again!

**Comment by spboyer** [comment]
@scottaddie - update your fork from master, you have some unicode characters that are messing up the build even in unrelated portions.

**Comment by Rick-Anderson** [comment]
Change it from the usually correct c# to none. It won't lex C# 5.

**Comment by scottaddie** [comment]
Sorry for the delay. I was sick for a few days. @Rick-Anderson's suggested change seemed to do the trick.

**Comment by Rick-Anderson** [comment]
Looks good to me. @spboyer  do you want to take a quick look?

**Comment by spboyer** [comment]
Thanks Scott for making the update and Rick for the assist.

**Pull request merged**